api/serializers.py

```diff
@@ -1,36 +1,27 @@
-from django.contrib.auth.models import User
 from rest_framework import serializers
 
 from .models import Profile, Video
 
 
 class ProfileSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = Profile
-        fields = "__all__"
-
-
-class UserSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = User
-        fields = "__all__"
-
-
-class UserRegistrationSerializer(serializers.ModelSerializer):
     def create(self, validated_data):
-        user = User.objects.create_user(**validated_data)
-        user.email = user.username
-        user.save()
-
-        # A profile with default data is created here. We can add more as needed
-        # in the profile model.
-        user.profile = Profile(user.id)
-        user.profile.save()
-        return user
+        profile = Profile.objects.create_user(
+            **validated_data, email=validated_data["username"]
+        )
+        return profile
 
     class Meta:
-        model = User
-        fields = "__all__"
+        model = Profile
+        fields = (
+            "id",
+            "username",
+            "email",
+            "first_name",
+            "last_name",
+            "phone_number",
+            "age",
+            "gender",
+        )
```

Comment:
Nice

Reply:
I was also going off of [DRF's recommendation](https://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include). I'm favoring being explicit about what we want to get from the serializer.